src/main/java/com/microsoft/sqlserver/jdbc/dns/DNSUtilities.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Microsoft JDBC Driver for SQL Server
+ * 
+ * Copyright(c) Microsoft Corporation All rights reserved.
+ * 
+ * This program is made available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.jdbc.dns;
+
+import java.util.Hashtable;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.naming.NamingEnumeration;
+import javax.naming.NamingException;
+import javax.naming.directory.Attribute;
+import javax.naming.directory.Attributes;
+import javax.naming.directory.DirContext;
+import javax.naming.directory.InitialDirContext;
+
+public class DNSUtilities {
+
+    private final static Logger LOG = Logger.getLogger(DNSUtilities.class.getName());
+
+    private static final Level DNS_ERR_LOG_LEVEL = Level.FINE;
+
+    /**
+     * Find all SRV Record using DNS.
+     *
+     * You can then use {@link DNSRecordsSRVCollection#getBestRecord()} to find the best candidate (for instance for Round-Robin calls)
```

Comment:
Hello @pierresouchay, thank you very much for your contribution. Just one question and wish you could help us. This line of JavaDoc gives error to us,  because Maven cannot find a reference to `DNSRecordsSRVCollection#getBestRecord()`, could you please let us know how we can fix it? Thank you.